runtime/nls/verb/verbose.nls

```diff
@@ -49,7 +49,7 @@ J9NLS.HEADER=verbosenls.h
 J9NLS_VERB_STACK_USAGE=Verbose stack: \"%2$.*1$s\" used %3$zd/%4$zd bytes on Java/C stacks
 # START NON-TRANSLATABLE
 J9NLS_VERB_STACK_USAGE.sample_input_1=11
-J9NLS_VERB_STACK_USAGE.sample_input_2=Gc Slave Thread
+J9NLS_VERB_STACK_USAGE.sample_input_2=Gc Worker Thread
```

Comment:
@SueChaplain fyi. Does any doc need to be modified or regenerated after these changes?

Reply:
I don't think any docs contain sample input.